samples/matter/light_bulb/src/aws_iot_integration/certs/ca-cert.pem

```diff
@@ -1,3 +1,3 @@
-"-----BEGIN CERTIFICATE-----\n"
-"-----certificate-----\n"
-"-----END CERTIFICATE-----\n"
+-----BEGIN CERTIFICATE-----
```

Comment:
Since this is a mandatory file for the application to work, I suggest we delete this file in the repo so that user gets a Cmake error when generating the include file.

Reply:
Or perhaps its only mandatory if `CONFIG_MQTT_HELPER_PROVISION_CERTIFICATES` is set?

Reply:
Makes sense to not include dummies, the sample wont build out of the box but we can include dummies in the sample test case to make it.

Reply:
Actually, I was mistaken. The current implementation allows for this. Maybe its fine that it builds even though the certs arent included? Do we need to throw an error? We could add some cmake magic here.

cc @balaji-nordic

Reply:
Discussed offline. I am fine leaving it as is. 
Reasons: 
- There will be a run time error and that is good enough to warn developers.
- If we remove these certificates, then `sample.yaml` needs significant updates to make it compile the application in CI. (The certs have to be generated when coming a configuration for the sake of CI)
- This is how it has been for a while now and we did not hear negative feedback or user mistakes related to this.

Reply:
After a bit discussion, we decide to remove the .pem files anyway. The behavior of samples will remain the same. There wont be an error during compilation but it will be a run time error just like it is now.